tests/models/test_hparams.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+import pytest
+import torch
+
+from pytorch_lightning import Trainer
+from pytorch_lightning.core.lightning import CHECKPOINT_KEY_MODULE_ARGS
+from tests.base import EvalModelTemplate
+
+
+class SubClassEvalModel(EvalModelTemplate):
+    object_that_should_not_be_saved = torch.nn.CrossEntropyLoss()
+
+    def __init__(self, *args, subclass_arg=1200, **kwargs):
```

Comment:
@Borda where do we test for:

```
GAN(Encoder(), Decoder(), nn.CrossEntropy(), learning_rate=1e-2)
```

Reply:
added, just instead of module as an argument I used customer loss